src/Internal/Wallet/Cip30Mock.purs

```diff
@@ -210,3 +274,48 @@ mkCip30Mock pKey mSKey = do
 
 -- returns an action that removes the mock.
 foreign import injectCip30Mock :: String -> Cip30Mock -> Effect (Effect Unit)
+
+-- Helpers
+
+foreign import raisePaginateError :: forall a. Int -> Effect a
```

Comment:
kind annotations missing for foralls

Reply:
Why there should be kind annotation? Looks like regular Type, no?

Reply:
It's unfortunately something from MLabs code style we inherited. It should be `forall (a :: Type). `